src/app/legacy/containers/PageHandlers/withHashChangeHandler/isHashChangeOnSamePath.js

```diff
@@ -1,6 +1,6 @@
 export default (
-  { location: { hash: prevHash, pathname: prevPathname } },
-  { location: { hash: nextHash, pathname: nextPathname } },
+  { location: { hash: prevHash, pathname: prevPathname } = {} },
+  { location: { hash: nextHash, pathname: nextPathname } = {} },
```

Comment:
When changing the service in storybook, I would get the following error:

![Screenshot 2024-02-02 at 11 19 12](https://github.com/bbc/simorgh/assets/58214768/a726ad5d-0a32-45e9-ad76-78d052b1525d)

Storybook uses hash links, but in this case, the 2 params to this function were set to `undefined` causing the page to crash. This should have minimal impact on the application (plus it's Single Page Application (SPA) functionality which we don't use (I think?))